portal_pages/templates/portal_pages/home_page.html

```diff
@@ -33,7 +33,9 @@
         <div class="page-width cushion" id="homepage-video">
             <h2>{{ self.youtube_video_title }}</h2>
             {{ self.youtube_blurb|richtext }}
-            <div id="homepage-video-player"></div>
+            <div class="responsive-object" style="padding-bottom: 56.25%;">
```

Comment:
wagtail does not set this padding-bottom to a fixed value but rather to a ratio, see: https://github.com/torchbox/wagtail/blob/master/wagtail/wagtailembeds/format.py#L8 

We should do the same, either by using their underlying support for doing this kind of embed or replicating what they are doing to compute the ratio....